### PR TITLE
*: move external storage to dumpling and lightning docs

### PR DESCRIPTION
**Pull request by shichun-0415**
### What is changed, added or deleted? (Required)
After BR refactor in https://github.com/pingcap/docs-cn/pull/11316, external storage is replaced by [Backup Storages](https://docs.pingcap.com/tidb/stable/backup-and-restore-storages). However, because some description about Dumpling and Lightning are still be useful and referenced by Dumpling and Lightning docs, external storage is removed from the navigation but kept in the docs repo. It would be better to move information about Dumpling and Lightning to the docs of these two tools, instead of being wrapped in br docs.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: dumpling-overview.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ dumpling -u root -P 4000 -h 127.0.0.1 --filetype sql -t 8 -o /tmp/test -r 200000
 以上命令中：
 
 - `-h`、`-P`、`-u` 分别代表地址、端口、用户。如果需要密码验证，可以使用 `-p $YOUR_SECRET_PASSWORD` 将密码传给 Dumpling。
-- `-o` 用于选择存储导出文件的目录，支持本地文件路径或[外部存储 URL](/br/external-storage.md) 格式。
+- `-o` （或 `--output`）用于选择存储导出文件的目录，支持本地文件路径或[外部存储 - URL 格式](/br/backup-and-restore-storages.md#url-格式) 格式。
```
Comment (Oreoxmt) [inline dumpling-overview.md:70]:
```suggestion
- `-o`（或 `--output`）用于选择存储导出文件的目录，支持本地文件路径或[外部存储 URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```

File: dumpling-overview.md [hunk 2]
```diff
@@ -334,7 +334,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | -s 或--statement-size | 控制 `INSERT` SQL 语句的大小，单位 bytes |
 | -F 或 --filesize | 将 table 数据划分出来的文件大小，需指明单位（如 `128B`, `64KiB`, `32MiB`, `1.5GiB`） |
 | --filetype| 导出文件类型（csv/sql） | "sql" |
-| -o 或 --output | 导出本地文件路径或[外部存储 URL](/br/external-storage.md) | "./export-${time}" |
+| -o 或 --output | 导出本地文件路径或[外部存储 - URL 格式](/br/backup-and-restore-storages.md#url-格式) | "./export-${time}" |
```
Comment (Oreoxmt) [inline dumpling-overview.md:337]:
```suggestion
| -o 或 --output | 导出本地文件路径或[外部存储 URL 格式](/br/backup-and-restore-storages.md#url-格式) | "./export-${time}" |
```

File: dumpling-overview.md [hunk 3]
```diff
@@ -182,7 +182,7 @@ export AWS_ACCESS_KEY_ID=${AccessKey}
 export AWS_SECRET_ACCESS_KEY=${SecretKey}
 ```
 
-Dumpling 同时还支持从 `~/.aws/credentials` 读取凭证文件。更多 Dumpling 存储配置可以参考[外部存储](/br/backup-and-restore-storages.md)。
+Dumpling 同时还支持从 `~/.aws/credentials` 读取凭证文件。Dumpling 导出到 Amazon S3 的配置参数与 BR 大致相同，更多参数描述，请参考[外部存储 - URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```
Comment (Oreoxmt) [inline dumpling-overview.md:185]:
```suggestion
Dumpling 同时还支持从 `~/.aws/credentials` 读取凭证文件。Dumpling 导出到 Amazon S3 的配置参数与 BR 大致相同，更多参数描述，请参考[外部存储 URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```

File: sql-statements/sql-statement-backup.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ BR 支持备份数据到 Amazon S3 或 Google Cloud Storage (GCS)：
 BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/?access-key={YOUR_ACCESS_KEY}&secret-access-key={YOUR_SECRET_KEY}';
 ```
 
-有关详细的 URL 语法，见[外部存储](/br/external-storage.md)。
+有关详细的 URL 语法，见[外部存储 - URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-backup.md:109]:
```suggestion
有关详细的 URL 语法，见[外部存储 URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```

File: sql-statements/sql-statement-restore.md [hunk 5]
```diff
@@ -101,7 +101,7 @@ BR 支持从 Amazon S3 或 Google Cloud Storage (GCS) 恢复数据：
 RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/';
 ```
 
-有关详细的 URL 语法，见[外部存储](/br/external-storage.md)。
+有关详细的 URL 语法，见[外部存储 - URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-restore.md:104]:
```suggestion
有关详细的 URL 语法，见[外部存储 URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```

File: tidb-lightning/tidb-lightning-data-source.md [hunk 6]
```diff
@@ -329,6 +329,31 @@ type = '$3'
 - **key**：文件的序号，即前文所述`${db_name}.${table_name}.001.csv`中的`001`。
     - 正则表达式匹配到的 group 序号，例如 “$4”。
 
+## 从 Amazon S3 导入数据
+
+如下为从 Amazon S3 导入数据的示例，更多配置参数描述，可参考[外部存储 -URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-data-source.md:334]:
```suggestion
如下为从 Amazon S3 导入数据的示例，更多配置参数描述，可参考[外部存储 URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 7]
```diff
@@ -104,7 +104,14 @@ backend = "local"
...
 
-如果数据源存放在 Amazon S3 或 GCS 等外部存储中，请参考[外部存储](/br/external-storage.md)。
+如果数据源存放在 Amazon S3 或 GCS 等外部存储中，需要额外的连接配置，你可以为这类配置指定参数。如下例子假设数据源存放在 Amazon S3 中：
+
+```
+./tidb-lightning --tidb-port=4000 --pd-urls=127.0.0.1:2379 --backend=local --sorted-kv-dir=/tmp/sorted-kvs \
+    -d 's3://my-bucket/sql-backup'
+```
+
+更多参数设置，请参考[外部存储 - URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-distributed-import.md:114]:
```suggestion
更多参数设置，请参考[外部存储 URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 8]
```diff
@@ -136,7 +143,7 @@ nohup tiup tidb-lightning -config tidb-lightning.toml > nohup.out &
 
 ## 示例 2：使用 TiDB Lightning 并行导入单表数据
 
-TiDB Lightning 也支持并行导入单表的数据。例如，将存放在 Amazon S3 中的多个单表文件，分别由不同的 TiDB Lightning 实例并行导入到下游 TiDB 数据库中。该方法可以加快整体导入速度。关于更多远端存储信息，请参考 [TiDB Lightning 支持的远端存储](/br/external-storage.md)。
+TiDB Lightning 也支持并行导入单表的数据。例如，将存放在 Amazon S3 中的多个单表文件，分别由不同的 TiDB Lightning 实例并行导入到下游 TiDB 数据库中。该方法可以加快整体导入速度。TiDB Lightning 使用远端存储时的配置参数与 BR 大致相同，关于详细的参数配置，可以参考 [外部存储 - URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-distributed-import.md:146]:
```suggestion
TiDB Lightning 也支持并行导入单表的数据。例如，将存放在 Amazon S3 中的多个单表文件，分别由不同的 TiDB Lightning 实例并行导入到下游 TiDB 数据库中。该方法可以加快整体导入速度。TiDB Lightning 使用云端存储时的配置参数与 BR 大致相同，关于详细的参数配置，可以参考[外部存储 URL 格式](/br/backup-and-restore-storages.md#url-格式)。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**